src/api/world.py

```diff
@@ -57,6 +66,19 @@ def save(self) -> None:
         """
         raise NotImplementedError()
 
+    def get_blockstate(self, blockstate: str) -> Block:
```

Comment:
This function should return the string of the Amulet block rather than the Block class itself. If we'll change the Block class, it will require us to change all of our implementations of the "get_blockstate" function.